### PR TITLE
Add indicator badge when deleting service instance

### PR DESCRIPTION
**Pull request by el-mapache**
Previously, we didn't give the user any feedback after they clicked the
'confirm delete' button.

Handy gif reference:
delete-service-instance

### CODE REVIEW

File: static_src/components/service_instance_table.jsx [hunk 1]
```diff
@@ -22,10 +20,13 @@ function stateSetter() {
     serviceInstances,
     currentSpaceGuid,
     loading: ServiceInstanceStore.loading,
-    empty: !ServiceInstanceStore.loading && !serviceInstances.length
+    empty: !ServiceInstanceStore.loading && !serviceInstances.length,
+    updating: ServiceInstanceStore.updating
   };
 }
 
+const propTypes = {};
```
Comment (jcscottiii) [inline static_src/components/service_instance_table.jsx]: put this after the imports. (at the top)

**Comment by jcscottiii** [comment]
we need a way to handle the error case of deleting the service. (should turn off _updating). (it's a lot of stuff to add so if you want to make a separate PR, you can do that.) let me know your thoughts.

**Comment by el-mapache** [comment]
@jcscottiii I can add a test to the actions that verifies that the same method is from both `then` and `catch` outcomes of the promise. That will be pretty simple.

We don't really have any kind of error handling around this action, so I would definitely want to have any of that work in a separate ticket/PR

**Comment by jcscottiii** [comment]
@el-mapache that would be really good!

**Comment by el-mapache** [comment]
@jcscottiii Test file updated!

**Comment by jcscottiii** [comment]
@el-mapache could you make a card for taking care of the error case?

**Comment by el-mapache** [comment]
@jcscottiii Definitely!

**Pull request merged**